### ISSUE TITLE
Support express v5.x

### ISSUE THREAD

**Issue by dsine-de** [szymmis/vite-express#112]
Hi!

Will there be support for the 5.x versions of Express?
I just tried the beta-1 but got an error:

```
TypeError: Cannot read properties of undefined (reading 'stack')
    at <...>\node_modules\vite-express\dist\main.js:152:49
    at Generator.next (<anonymous>)
    at fulfilled (<...>\node_modules\vite-express\dist\main.js:28:58)
```

**Comment by szymmis** [comment]
Hi @dsine-de.

Yes, express 5 will definitely be supported but I haven't really looked into it yet and don't know what structural changes will be needed to achieve the support.

Thanks for letting me know about 
the problem!

**Comment by elturpin** [comment]
I found the solution to the problem. As said here https://expressjs.com/en/guide/migrating-5.html#app.router the `router` object come back in version 5 as a public property.

You then just need `sed -i 's/_router/router/g' src/main.ts` and it will be good.

**Comment by szymmis** [comment]
Thank you very much @elturpin for figuring out where the problem lies in. 
That made making the change really easy for me.

Fix for that was released in v0.15.0 @dsine-de so you can check it out!
